code/datums/components/toggle_hood.dm

```diff
@@ -22,11 +23,15 @@
 
 /datum/component/toggle_hood/proc/flip_hood(atom/movable/thing, mob/user)
 	src.hooded = !src.hooded
-	suit.over_hair = !suit.over_hair
-	suit.body_parts_covered ^= HEAD
+	src.suit.hooded = src.hooded
+
 	if (ismob(suit.loc))
 		var/mob/M = suit.loc
+		if (!isliving(M))
+			suit.body_parts_covered ^= HEAD
+			suit.over_hair = !suit.over_hair
```

Comment:
not `ishuman()`?

Reply:
Was trying to preserve existing logic but sure